doc/alertmanager-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,310 @@
+local grafana = import 'github.com/grafana/grafonnet-lib/grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+
+
```

Comment:
You have a bunch of those double-blank-line gaps in this file. I could see a reason for them right here, but further down they seem fairly random. My apologies if there is an idea behind it that I didn't see. It's just a nit, but generally, I prefer minimizing vertical whitespace, i.e. sure, use a blank line for structuring, but _two_ blank lines probably doesn't really carry additional meaning for most, but takes away screen real estate.